### ISSUE TITLE
Remove/update accessibility_tools dependency

### ISSUE THREAD

**Issue by alestiago** [widgetbook/widgetbook#1197]
**Description**

This item is to track the removal/update of the `accessibility_tools` dependency.

It seems that from https://github.com/widgetbook/widgetbook/pull/1193 the `package:accessibility_tools` dependency will no longer be used by `package:widgetbook`.

Currently, if you follow the [migration guide](https://docs.widgetbook.io/addons/accessibility-addon) and try using the latest `package:accessibility_tools` version ([2.1.0](https://pub.dev/packages/accessibility_tools/changelog)) you will be unable (unless overridden) to do so because `package:widgetbook` constraints it to:

https://github.com/widgetbook/widgetbook/blob/6a024a7f792f3eee20847e6f605bf812c60cbdda/packages/widgetbook/pubspec.yaml#L19

**Expected Behavior**
Users should be able to use the latest version of `package:accessibility_tools` gracefully.

**Additional context**

- I'm creating this issue to track the effort. If there is an existing issue (I couldn't find one) feel free to close in favour of such.

**Comment by YoussefRaafatNasry** [comment]
Hello @alestiago 👋 

Currently we have decided to not provide first-class support for `accessibility_tools` as the package gets frequent updates, especially with every new Flutter release. This frequent updates is affecting our own [package score](https://pub.dev/packages/widgetbook/score). 

We can keep the package's version unconstrained, but this can lead to having less control over our own package and its backwards compatibility. Thus, we have decided to let the users setup the addon themselves, which is not a big of a hassle.

Yet, your point about the migration guide is true, and it definitely needs an update there to either constraint the version or provide a guide for overriding the version.

**Comment by alestiago** [comment]
Hi @YoussefRaafatNasry thanks for the quick reply! 

Will the `accessibility_tools` dependency get removed once the deprecation cycle of the `AccessibilityAddon` ends?

**Comment by YoussefRaafatNasry** [comment]
Yes, it will be completely removed after the deprecation ends. This might take some time tho, as it will be with the next major release.

We only used the package for this addon, so it will be safe to remove.

**Comment by yousinix** [comment]
This issue is no longer relevant since we bumped it to v2 in #1253
